vscode-extension/package.json

```diff
@@ -128,10 +128,12 @@
     "@eclipse-sirius/sirius-components-diagrams": "*",
     "@eclipse-sirius/sirius-components-forms": "*",
     "@eclipse-sirius/sirius-components-selection": "*",
+    "@eclipse-sirius/sirius-components-diagrams-reactflow": "*",
```

Comment:
All the peer dependencies should appear as dependencies here so react-flow should be listed here. To be honest there seems to be a lot of missing dependencies:

```
    "@material-ui/core": "4.12.4",
    "@material-ui/icons": "4.11.3",
    "@material-ui/lab": "4.0.0-alpha.61",
    "@xstate/react": "1.6.3",
    "lexical": "0.8.1",
    "d3": "7.0.0",
    "html-to-image": "1.11.11",
    "reactflow": "11.8.1",
    "reflect-metadata": "0.1.13",
    "sprotty": "0.12.0",
    "xstate": "4.32.1"
```

I'm not sure why that works today as it should explicitly not

Reply:
Added.